main/templates/components/card-list.html

```diff
@@ -1,33 +1,40 @@
 {% load static %}
 <div id="card-list-item-{{object.id}}" class="container-fluid card-list mw-100 px-2 card-list rounded py-1" tabindex="1">
-  <span class="object-id d-none">{{ object.id }}</span>
+  <span class="object-id  d-none">{{ object.id }}</span>
+   
+
       
     <div class="row">
       <div class="col-lg-4 col-6">
         <strong><h5 title="{{object.surname}}, {{object.name}}">{{object.surname}}, {{object.name}}</h5></strong>
       
       </div>
-      <div class="col-lg-3 col-6">
+      <div class="col-city col-md-2 col-lg-2 col-sm-2">
+
         {% if object.city %}
         <span>{{object.city}}</span>
         {% else %}
         <span>No especificado</span>
         {% endif %}
 
       </div>  
-      <div class="col-lg-2 col-6">
+
+      <div class="col-telephone col-md-3 col-lg-2  col-sm-1">
+
         {% if object.telephone %}
         <span>{{object.telephone}}</span>
         {% else %}
         <span>No especificado</span>
         {% endif %}
 
       </div> 
-      <div class="col-lg-3 col-6">
+
+      <div class="col-email col-md-2 col-sm-1 col-lg-2 col-sm-1 col-email ">
+
         {% if object.email %}
         <span>{{object.email}}</span>
         {% else %}
-        <span>No especificado</span>
+        <span>No especificado</spa.n>

```

Comment:
There is a typo here
![image](https://user-images.githubusercontent.com/53529939/231481615-39448117-7e03-4341-a97d-a1dc0f723ea9.png)